lib/internal/test_runner/test.js

```diff
@@ -53,30 +32,36 @@ const kSubtestsFailed = 'subtestsFailed'
 const kTestCodeFailure = 'testCodeFailure'
 const kDefaultIndent = '    '
 const noop = FunctionPrototype
+const isTestRunner = getOptionValue('--test')
```

Comment:
Interesting, I had this always `true` before, maybe this also explains test failures